src/components/HabitList.js

```diff
@@ -1,15 +1,22 @@
 import React, { Component } from 'react'
+import styled from 'react-emotion'
+
+import Headline from '../styles/Headline'
 
 import CountButton from './CountButton'
 import ToggleButton from './ToggleButton'
-import List from '../styles/List'
+
+const StyledDiv = styled('div')`
+  width: 95%;
```

Comment:
ungewöhnlich. Lieber hier `width: 100%` (also ganz weg lassen) und in den Buttons `width: auto`. Das möchtest du wahrscheinlich eher, dann ist es nicht so nach rechts versetzt.